sycl/doc/GetStartedWithSYCLCompiler.md

```diff
@@ -259,6 +282,9 @@ ninja check-all
 If no OpenCL GPU/CPU runtimes are available, the corresponding tests are
 skipped.
 
+If CUDA support has been built, it is tested only if there are devices 
```

Comment:
only if there are devices available that provide support for CUDA.

Reply:
Addressed in 7d027efeda7845a8d6df6f7fa109d4c73451ea61